commons/src/destination.rs

```diff
@@ -0,0 +1,54 @@
+use core::fmt;
+use crate::runtime::AccountId;
+
+pub type BtcTransactionId = [u8; 32];
+pub type BankTransactionId = [u8; 32];
+
+/// Identifier for different destinations on VLN
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Eq,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    parity_scale_codec::Decode,
+    parity_scale_codec::Encode,
+)]
+pub enum Destination {
+    // type to denote destinations on vln blockchain
+    Vln(AccountId),
+    // type to denote transactions on the bitcoin blockchain
+    Btc(BtcTransactionId),
```

Comment:
This was mostly to give an example so I'd leave it out for now since we are not likely to support BTC destinations in the time being.